packages/applications-service-api/src/controllers/subscriptions.js

```diff
@@ -78,6 +78,7 @@ const getApplicationOrThrow = async (caseReference) => {
 
 const validateSubscriptionDate = (subscriptionCreatedAt) => {
 	const expiryTime = moment(subscriptionCreatedAt).add(48, 'hours');
+	console.log({ expiryTime, now: moment() });
```

Comment:
is this still needed?

Reply:
Sorry I thought I removed this - will remove now.